web3.js/src/transaction-constants.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Maximum over-the-wire size of a Transaction
+ *
+ * 1280 is IPv6 minimum MTU
+ * 40 bytes is the size of the IPv6 header
+ * 8 bytes is the size of the fragment header
+ */
+export const PACKET_DATA_SIZE = 1280 - 40 - 8;
+
+export const SIGNATURE_LENGTH = 64;
```

Comment:
Can you add docs that state this length is measured in bytes (as opposed to base58 string length)